README.md

```diff
@@ -49,8 +49,8 @@ Node.js client library to use the Watson APIs.
 </details>
 
 ## ANNOUNCEMENTS!
-### New Major Version
-The new major version, `v4` has been released. The major new feature in this release is native support for Promises. There are some breaking changes - use the [migration guide](https://github.com/watson-developer-cloud/node-sdk/blob/master/UPGRADE-4.0.md) to navigate any changes that affect your code.
+### Deprecating support for Node versions 6 and 8
```

Comment:
I think that we can just say that we support Node > 10. No need to say that we deprecated the support for an older version unless we know it won't work

Reply:
Sounds good to me 👍